src/registrar/fixtures.py

```diff
@@ -0,0 +1,256 @@
+import logging
+import random
+import string
+from faker import Faker
+
+from registrar.models import (
+    User,
+    UserProfile,
+    DomainApplication,
+    Domain,
+    Contact,
+    Website,
+)
+
+fake = Faker()
+logger = logging.getLogger(__name__)
+
+
+class UserFixture:
+    """
+    Load users into the database.
+
+    Make sure this class' `load` method is called from `handle`
+    in management/commands/load.py, then use `./manage.py load`
+    to run this code.
+    """
+
+    ADMINS = [
+        {
+            "username": "c4a0e101-73b4-4d7d-9e5e-7f19a726a0fa",
+            "first_name": "Seamus",
+            "last_name": "Johnston",
+        },
+        {
+            "username": "d4c3bd84-dc3a-48bc-a3c3-f53111df2ec6",
+            "first_name": "Igor",
+            "last_name": "",
+        },
+        {
+            "username": "ee80bfe0-49ad-456d-8d82-e2b608a66517",
+            "first_name": "Logan",
+            "last_name": "",
+        },
+    ]
```

Comment:
@michelle-rago @boonerang @calerubensteingsa @neilmb @h-m-f-t 

Currently refactoring (and expanding) our data fixtures and thought I'd ask again if any of you would care to have your account ID from the Login.gov sandbox included here.

Also, if you want some custom fixture data (beyond being an admin and having a few example domain applications), I'm happy to take requests. CC @loganmeetsworld  @igorkorenfeld on that last sentence.

Reply:
@SSPJ What does adding our sandbox account here get us?

Reply:
I'm good with what is there for me.

Reply:
@michelle-rago Unsure if you are asking what _this file_ does currently or what _data fixtures_ in general do, so answering both.

**Data fixtures:** We periodically have to recreate the database in staging or unstable. When we do that, we can run the code in this file to generate data. If you'd like to have fake data associated with your user, this is where we'd put it. An example might be a page that you can only access once you have a domain approved: if you'd like to check the wording on that page, you'd need to create an application, submit it, then approve it. A data fixture speeds the process by giving you an already approved domain.

**Currently:** The `DomainApplicationFixture` (further down in the file) randomly generates about 4 applications per user, in various states. If you have more specific requests, I'm happy to add them!